docs/content/tutorials/ingress-path-regex-annotation.md

```diff
@@ -0,0 +1,457 @@
+---
+title: Customizing Ingresses with Path-Regex Annotations
+description: |
+  How to customize Simple Ingress and Mergeable Ingress types with path-regex annotations.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+---
+## Customizing NGINX Ingress Controller with Path-Regex Annotations
+
+This document explains how to use `path-regex` annotations with Simpe Ingress and Mergeable Ingress (Master and Minions)
```

Comment:
This could simply be:
"This document explains how to use `path-regex` annotation with the Ingress resource"
Calling out the use cases is not necessary to the reader deciding if they continue.
"Simple" means something to us, but it is not apparent to a reader what that means.  A better section header might be "basic use" 
Master/minion is immediately recognizable and a good section header.

Reply:
"Simple" if keeping this phrasing.